app/src/ui/check-runs/ci-check-run-popover.tsx

```diff
@@ -217,8 +217,18 @@ export class CICheckRunPopover extends React.PureComponent<
     this.props.dispatcher.openInBrowser(url)
   }
 
-  private markDownLinkClicked = (link: string): void => {
-    this.props.dispatcher.openInBrowser(link)
+  private onViewJobStep = (
+    checkRun: IRefCheck,
+    step: IAPIWorkflowJobStep
+  ): void => {
+    const url = checkRun.htmlUrl
+      ? `${checkRun.htmlUrl}/#step:${step.number}:1`
+      : `${this.props.repository.htmlURL}/pull/${this.props.prNumber}`
+    if (url === null) {
+      // The repository should have a htmlURL.
+      return
+    }
```

Comment:
This is impossible, right? Since we're using template strings, `url` can't be null.
```suggestion
```

Reply:
Yep. I think that null check was left over from a different approach. Thanks! I updated both instances of using this.